rpcs3/Emu/RSX/VK/VulkanAPI.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#define VK_USE_PLATFORM_WIN32_KHR
```

Comment:
BTW currently Vulkan support is really sparse on Linux. It's supported by Skylake gpu (older Intel cpu support is apparently incomplete) on Wayland (Xlib binding was mentionned as untested in the LunarG SDK, although I'm expecting it to work somehow) and with Nvidia with XLib. AMD Vulkan support is Windows only and atm Android support is non existent.